hack/setup-dev-environment.sh

```diff
@@ -50,11 +50,22 @@ nodes:
     apiServer:
       extraArgs:
         "service-node-port-range": "1-65535"
+  - |
+    kind: InitConfiguration
+    nodeRegistration:
+      kubeletExtraArgs:
+        node-labels: "ingress-ready=true"
   extraPortMappings:
   - containerPort: ${REGISTRY_PORT:-5000}
     hostPort: ${REGISTRY_PORT:-5000}
     listenAddress: 127.0.0.1
     protocol: TCP
+  - containerPort: 80
+    hostPort: 80
```

Comment:
It would be nice to not have to use a low port number here.

Reply:
Yeah, perhaps in a followup, this would involve plugins, not sure if these are configurable via Helm charts...